test/integration/rippled.cfg

```diff
@@ -0,0 +1,963 @@
+#
+#   This port is commented out but can be enabled by removing
+#   the '#' from each corresponding line including the entry under [server]
+#
+#   "wss_public"
+#
+#       Guest level API commands over Secure Websockets, open to everyone.
+#
+#   For HTTPS and Secure Websockets ports, if no certificate and key file
+#   are specified then a self-signed certificate will be generated on startup.
+#   If you have a certificate and key file, uncomment the corresponding lines
+#   and ensure the paths to the files are correct.
+#
+#   NOTE
+#
+#       To accept connections on well known ports such as 80 (HTTP) or
+#       443 (HTTPS), most operating systems will require rippled to
+#       run with administrator privileges, or else rippled will not start.
+
+[server]
+port_rpc_admin_local
+#port_peer
+port_ws_admin_local
+#port_ws_public
+#port_ws_admin_public
+#ssl_key = /etc/ssl/private/server.key
+#ssl_cert = /etc/ssl/certs/server.crt
+
+[port_rpc_admin_local]
+port = 5005
+ip = 127.0.0.1
+admin = 127.0.0.1
+protocol = http
+
+[port_peer]
+port = 51235
+ip = 0.0.0.0
+protocol = peer
+
+[port_ws_admin_local]
+port = 6006
+# ip = 127.0.0.1
+# admin = 127.0.0.1
+ip = 0.0.0.0
+admin = 0.0.0.0
+protocol = ws
+
+[port_ws_admin_public]
+port = 5007
+ip = 0.0.0.0
+admin = 192.168.50.1
+protocol = ws
+
+[port_ws_public]
+port = 5006
+ip = 0.0.0.0
+protocol = ws
+
+#[port_ws_public]
+#port = 5005
+#ip = 127.0.0.1
+#protocol = wss
+
+#-------------------------------------------------------------------------------
+
+[node_size]
+medium
+
+# This is primary persistent datastore for rippled.  This includes transaction
+# metadata, account states, and ledger headers.  Helpful information can be
+# found here: https://ripple.com/wiki/NodeBackEnd
+# delete old ledgers while maintaining at least 2000. Do not require an
+# external administrative command to initiate deletion.
+[node_db]
+type=RocksDB
+path=/var/lib/rippled/db/rocksdb
+open_files=2000
+filter_bits=12
+cache_mb=256
+file_size_mb=8
+file_size_mult=2
+online_delete=2000
+advisory_delete=0
+
+[database_path]
+/var/lib/rippled/db
+
+# This needs to be an absolute directory reference, not a relative one.
+# Modify this value as required.
+[debug_logfile]
+/var/log/rippled.debug.log
+
+[sntp_servers]
+time.windows.com
+time.apple.com
+time.nist.gov
+pool.ntp.org
+
+# Where to find some other servers speaking the Ripple protocol.
+#
+#[ips]
+#r.ripple.com 51235
+
+# Public keys of the validators that this rippled instance trusts.  The latest
+# list of validators can be obtained from https://ripple.com/ripple.txt
+#
+# See also https://wiki.ripple.com/Ripple.txt
+#
+[validators]
+n949f75evCHwgyP4fPVgaHqNHxUVN15PsJEZ3B3HnXPcPjcZAoy7    RL1
+n9MD5h24qrQqiyBC8aeqqCWvpiBiYQ3jxSr91uiDvmrkyHRdYLUj    RL2
+n9L81uNCaPgtUJfaHh89gmdvXKAmSt5Gdsw2g1iPWaPkAHW5Nm4C    RL3
+n9KiYM9CgngLvtRCQHZwgC2gjpdaZcCcbt3VboxiNFcKuwFVujzS    RL4
+n9LdgEtkmGB9E2h3K4Vp7iGUaKuq23Zr32ehxiU8FWY7xoxbWTSA    RL5
```

Comment:
Is it actually using these?

Reply:
Based on the console log, it looks like it uses them:
```
2017-Nov-22 14:52:26 ValidatorList:DBG 5  of 5 listed validators eligible for inclusion in the trusted set
2017-Nov-22 14:52:26 ValidatorList:DBG Using quorum of 3 for new set of 5 trusted validators
```
Not sure though why it would need validators in the Standalone mode. Maybe someone from the rippled team can review the config.

Reply:
It definitely shouldn't be using them if it's in standalone mode...

Reply:
It does run in standalone mode according to the log:
`2017-Nov-22 16:29:17 Application:WRN Running in standalone mode`

Reply:
It looks like rippled needlessly populates its list of validators to trust and determines a quorum (thus the log messages).
However, the validator list and quorum are not used in standalone mode.

Reply:
thanks for clarification